doc/input-output-reference/src/overview/group-solar-collectors.tex

```diff
@@ -568,6 +568,117 @@ \subsubsection{Inputs}\label{inputs-5-029}
   0.84 ;                                                     !- Front Surface Emittance
 \end{lstlisting}
 
+\subsection{SolarCollectorPerformance:PhotovoltaicThermal:BIPVT}\label{solarcollectorperformancephotovoltaicthermalBIPVT}
+
+This object is used to provide performance details for the Building-Integrated PVT (BIPVT) model. There are no specific output variables for this object, reporting is done by the parent PVT object.
+
+
+\subsubsection{Inputs}\label{BIPVT-Inputs}
+
+\paragraph{Field: Name}\label{BIPVT-field-name}
+
+This field is the unique name for the BIPVT solar collector.
+
+\paragraph{Field: Boundary Conditions Model Name}\label{BIPVT-field-BC}
+
+This field contains the name of a \hyperref[surfacepropertyothersideconditionsmodel]{SurfaceProperty:OtherSideConditionsModel} object declared elsewhere in the input file. The “\textit{Type of Modelling}” for this object will be set to “\textit{GapConvectionRadiation}”. This will connect the collector to the exterior boundary conditions for the underlying heat transfer surface in the EnergyPlus building envelope model.
+
+\paragraph{Field: Availability Schedule Name}\label{BIPVT-field-Sched}
+
+This field contains the name of a schedule to indicate when the solar collector is available. When the schedule value is 0, the collector will be bypassed. When the value is greater than 0 the collector is available to provide heat recovery. If this field is left blank, it is assumed that the collector is available.
+
+\paragraph{Field: Effective Plenum Gap Thickness Behind PV modules}\label{BIPVT-field-Gap}
+
+This field is used to enter a nominal thickness of the air channel between the PV modules and the building. This is used to calculate the convective heat transfer coefficient behind the PV modules and on the building wall/roof surface. Units are \si{\m}. 
+
+\paragraph{Field: Effective Overall Height of Collector}\label{BIPVT-field-Height}
```

Comment:
This field and next were removed from the IDD, so they need to be removed here. Also some other field names need to be updated to match the IDD. Also check the idf snippet below.